src/pages/Index.vue

```diff
@@ -2,18 +2,115 @@
   <Layout>
 
     <!-- Learn how to use images here: https://gridsome.org/docs/images -->
-    <h1>Welcome to the Galaxy Mentoring Network</h1>
+    <g-image alt="Example image" src="~/favicon.png" width="135" />
```

Comment:
Why adding the logo again there as it is already in the navbar?

Reply:
I didn't add it. It came with the file before I started editing it, and because I couldn't preview the page locally, I feared taking out an important line, that's why I didn't touch it

Reply:
@Chinonyemildred Why couldn't you preview the page locally?

Reply:
there was an issue with running my command on my vscode. I downloaded an installer from the readme file link in the description of the error. @rahmot helped a great deal and it is working fine now :)